openshift/templates/build.bc.yaml

```diff
@@ -58,44 +67,4 @@ objects:
       type: Git
     strategy:
       type: Docker
-    triggers:
-      - type: ConfigChange
-      - type: ImageChange
-- apiVersion: build.openshift.io/v1
-  kind: BuildConfig
-  metadata:
-    annotations:
-      openshift.io/generated-by: OpenShiftNewBuild
-    labels:
-      app: ${NAME}-${SUFFIX}
-    name: ${NAME}-${SUFFIX}
-  spec:
-    nodeSelector: null
-    runPolicy: SerialLatestOnly
-    completionDeadlineSeconds: 1200 # 20 minutes
-    output:
-      to:
-        kind: ImageStreamTag
-        name: ${NAME}:${SUFFIX}
-    resources:
-      limits:
-        cpu: '2'
-        memory: 5Gi
-      requests:
-        cpu: '1'
-        memory: 1Gi
-    source:
-      git:
-        uri: ${GIT_URL}
-        ref: origin/${GIT_BRANCH}
-      type: Git
-    strategy:
-      dockerStrategy:
-        from:
-          kind: ImageStreamTag
-          name: python:3.6
-          namespace: openshift
-      type: Docker
-    triggers:
-        - type: ConfigChange
-        - type: ImageChange
+    triggers: []
```

Comment:
There's no need to keep the line around without any inputs.  Please remove 'triggers: [ ]'.

Reply:
This is how you disable all the triggers. If we remove this, there is gonna be a config change trigger as a default